pages/linux/hashcat.md

```diff
@@ -0,0 +1,32 @@
+# hashcat
+
+> Fast and advanced password recovery tool.
+> More information: <https://hashcat.net/hashcat/>.
```

Comment:
I can't find any useful information regarding the command on this website. What do you think about using <https://manned.org/hashcat>?

Reply:
fixed